src/test/scala/org/broadinstitute/dsde/firecloud/service/MethodConfigurationServiceSpec.scala

```diff
@@ -162,7 +174,7 @@ class MethodConfigurationServiceSpec extends FreeSpec with ScalaFutures with Sca
       "MethodNotAllowed error is returned" in {
         Put(validCopyFromRepoUrl) ~> sealRoute(routes) ~> check {
           status should equal(MethodNotAllowed)
-          responseAs[String] === "HTTP method not allowed, supported methods: GET"
+          errorReportCheck("FireCloud", MethodNotAllowed)
```

Comment:
This is correct - the 405 error checks in tests should never hit the downstream endpoints - they should be captured at the orchestration layer.